Resources/ArmTemplates/parameters.json

```diff
@@ -57,16 +42,16 @@
         "value": "__cosmosDbCollectionApprenticeshipReport__"
       },
       "providerServiceSettingsApiUrl": {
-        "value": "__providerServiceSettingsApiUrl__"
+        "value": "__providerApiUrl__"
```

Comment:
This is defined in a shared variable set, and the old variable was just a proxy for it. This eliminates the need for the proxy variable